aioshelly/rpc_device.py

```diff
@@ -58,22 +61,26 @@ async def create(
 
         return instance
 
-    def _on_notification(self, method, params=None):
+    def _on_notification(
+        self, method: str, params: dict[str, Any] | None = None
+    ) -> None:
         if params is not None:
             if method == "NotifyStatus":
-                self._status = dict(mergedicts(self._status, params))
+                self._status = (
+                    dict(mergedicts(self._status, params)) if self._status else None
+                )
```

Comment:
We can't reach here if `self._status` is `None` so it is better to assert it
```suggestion
                assert self._status
                self._status = dict(mergedicts(self._status, params))
```